config/ccm-example.yaml

```diff
@@ -9,7 +9,6 @@ check_schema: false
 work_model:
   name: AffineCombination
   parameters:
-    alpha: 1.0
```

Comment:
Should these have a new variable in the yaml, like `default_alpha` when it's not specified?

Reply:
Oh this one had been forgotten! Removing it right now. Thanks.